starter-code/function.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var opinions = [
+  "This is the best job ever",
+  "Satisfied",
+  "At least I get paid",
+  "I'm looking for another job",
+  "I don't want to answer"
+];
+
+
+function pickOpinion(){
+  return opinions[Math.floor(Math.random()*opinions.length)];
```

Comment:
Here you are hard coding the variable opinions. That means that you can't use another array than opinions. Here is not a big deal, but keep that in mind.